src/functions/toggleItemsVisibility.js

```diff
@@ -0,0 +1,15 @@
+let isVisible = true;
+
+const toggleItemVisibility = (list) => {
+  if(isVisible) {
+    document.querySelector("." + list + ">.list-of-features").style.maxHeight = "0px";
```

Comment:
I would suggest applying this styles conditionally to eliminate code duplication

Reply:
Done.